jbpm-test-coverage/src/test/java/org/jbpm/test/functional/timer/GlobalQuartzDBTimerServiceTest.java

```
@@ -332,24 +329,29 @@ public void testTimerRequiresRecoveryFlagSet() throws Exception {
         Map<String, Object> params = new HashMap<String, Object>();
         params.put("test", "john");
         ProcessInstance processInstance = ksession.startProcess("PROCESS_1", params);
-       
+
+        Connection connection = null;
+        Statement stmt = null;
         try {
-            Connection connection = ((DataSource)InitialContext.doLookup("jdbc/jbpm-ds")).getConnection();
-            Statement stmt = connection.createStatement();
+            connection = ((DataSource)InitialContext.doLookup("jdbc/jbpm-ds")).getConnection();
+            stmt = connection.createStatement();
 
             ResultSet resultSet = stmt.executeQuery("select REQUESTS_RECOVERY from QRTZ_JOB_DETAILS");
             while(resultSet.next()) {
                 boolean requestsRecovery = resultSet.getBoolean(1);
                 assertEquals("Requests recovery must be set to true", true, requestsRecovery);
             }
-            
-            stmt.close();
-            connection.close();
         } catch (Exception e) {
-            
+            logger.error("Error during test.", e);
```

+1, rather let the exception bubble up. Will fix it in a moment.